torch/jit/__init__.py

```diff
@@ -709,11 +709,14 @@ def trace(func,
                                            results diverge numerically for a known reason, such as operator fusion.
 
     Returns:
-        A ``ScriptModule`` object with a single ``forward()`` method containing the traced code.
-        When ``func`` is a ``torch.nn.Module``, the returned ``ScriptModule`` will have the same set of
-        sub-modules and parameters as ``func``.
+        if ``callable`` is ``nn.Module`` or ``forward()`` of ``nn.Module``, ``trace`` returns
+        a ``ScriptModule`` object with a single ``forward()`` method containing the traced code.
+        The returned ``ScriptModule`` will have the same set of sub-modules and parameters as the
+        original ``nn.Module``
+        if ``callable`` is a standalone function, ``trace`` returns ``torch.jit._C.Function``
```

Comment:
Missing period + upper-case letter? That would make it much more readable.

Reply:
👍 will fix